src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $cardBorderColor;
+  background: $cardBgColor;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__heading {
+    margin: 0 0 4px;
+    color: $CardFontColorDark;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: $CardFontColorLight;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rate {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-text {
+      margin: 0;
+      color: $CardFontColorDark;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      margin: 0;
+      color: $CardFontColorLight;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      margin: 0;
+      color: $CardFontColorDark;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    background-color: $buttonBgColor;
+    border-radius: 5px;
+    border: none;
```

Comment:
it's better to set a transparent 1px border here and change its color on hover

Reply:
there is no point